lab-nassir/gulpfile.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+gulp.task('hello', function(){
```

Comment:
If you're using a generic gulpfile in all of your labs, you should remove this task as it's not being used :P